### PR TITLE
MTM-49613 Remade all screenshots with new design. Removed redundant s…

### PR DESCRIPTION
**Pull request by eniosultan**
…creenshots.

### CODE REVIEW

**Comment by BeateRixen** [comment]
@eniosultan I left some comments. Some of the new images do not get rendered. @catarino you fixed this in another PR didn´t you? What was the issue?

**Comment by BeateRixen** [comment]
@eniosultan The image cockpit-dataexplorer-adddatapoint shows different different content now. Is this intended?

**Comment by BeateRixen** [comment]
@eniosultan We need the same changes for the German user guide. It will be sufficient to re-create the screenshots here, changes in the md files will be done automatically with the next translation round.

File: content/users-guide/cockpit-bundle/exports.md [hunk 1]
```diff
@@ -145,8 +140,6 @@ Click **Create** to create the new export schedule.
 
 The export schedule will be added to the export details.
 
-![Scheduled exports list](/images/users-guide/cockpit/cockpit-export-schedule-list.png)
-
 ##### Migration of scheduled exports
```
[the remark this answers is not in the document]
Comment (eniosultan) [reply content/users-guide/cockpit-bundle/exports.md:143]: Removed the screenshot, but IMO it doesn't make sense to add the message as well, as it is just a simple info message.

File: content/users-guide/cockpit-bundle/using-widgets.md [hunk 2]
```diff
@@ -48,14 +44,10 @@ Some of the widget require additional permissions in order to visualize the data
...
-
 By dragging the arrows on the bottom right corner of a widget, you can resize it.
 
 To edit the properties of a widget, click the cogwheel icon at the top right corner of the widget and from the context menu select **Edit**.
 
-<img src="/images/users-guide/cockpit/cockpit-dashboard-widget-menu.png" name="Edit widget"/>
-
 To remove a widget from a dashboard or report, click the cogwheel icon at the top right corner of the widget and from the context menu select **Remove**.
 
 Widgets can only be modified if the dashboard/report is unlocked. To lock/unlock it, use the toggle with the lock icon in the top menu bar.
```
[the remark this answers is not in the document]
Comment (eniosultan) [reply content/users-guide/cockpit-bundle/using-widgets.md:53]: done

**Review by BeateRixen — APPROVED** [review]

**Review by janhommes — APPROVED** [review]

**Pull request merged**